test/transmuxer/transmuxer_integration.js

```diff
@@ -58,247 +100,197 @@ describe('Transmuxer Player', () => {
+      expect(player.isLive()).toBe(false);
 
-    // Wait for the video to start playback.  If it takes longer than 10
-    // seconds, fail the test.
-    await waiter.waitForMovementOrFailOnTimeout(video, 10);
+      // Wait for the video to start playback.  If it takes longer than 10
+      // seconds, fail the test.
+      await waiter.waitForMovementOrFailOnTimeout(video, 10);
 
-    // Play for 15 seconds, but stop early if the video ends.  If it takes
-    // longer than 45 seconds, fail the test.
-    await waiter.waitUntilPlayheadReachesOrFailOnTimeout(video, 15, 45);
+      // Play for 15 seconds, but stop early if the video ends.  If it takes
+      // longer than 45 seconds, fail the test.
+      await waiter.waitUntilPlayheadReachesOrFailOnTimeout(video, 15, 45);
 
-    await player.unload();
-  });
-
-  it('raw MP3', async () => {
-    if (!MediaSource.isTypeSupported('audio/mp4; codecs="mp3"')) {
-      return;
-    }
-    await player.load('/base/test/test/assets/hls-raw-mp3/playlist.m3u8');
-    await video.play();
-    expect(player.isLive()).toBe(false);
-
-    // Wait for the video to start playback.  If it takes longer than 10
-    // seconds, fail the test.
-    await waiter.waitForMovementOrFailOnTimeout(video, 10);
-
-    // Play for 15 seconds, but stop early if the video ends.  If it takes
-    // longer than 45 seconds, fail the test.
-    await waiter.waitUntilPlayheadReachesOrFailOnTimeout(video, 15, 45);
-
-    await player.unload();
-  });
+      await player.unload();
+    });
 
-  it('raw AC3', async () => {
-    if (!MediaSource.isTypeSupported('audio/mp4; codecs="ac-3"')) {
-      return;
-    }
-    // This tests is flaky in some Tizen devices, so we need omit it for now.
-    if (shaka.util.Platform.isTizen()) {
-      return;
-    }
-    // It seems that AC3 on Edge Windows from github actions is not working
-    // (in the lab AC3 is working). The AC3 detection is currently hard-coded
-    // to true, which leads to a failure in GitHub's environment.
-    // We must enable this, once it is resolved:
-    // https://bugs.chromium.org/p/chromium/issues/detail?id=1450313
-    const chromeVersion = shaka.util.Platform.chromeVersion();
-    if (shaka.util.Platform.isEdge() &&
-        chromeVersion && chromeVersion <= 116) {
-      return;
-    }
-
-    await player.load('/base/test/test/assets/hls-raw-ac3/prog_index.m3u8');
-    await video.play();
-    expect(player.isLive()).toBe(false);
-
-    // Wait for the video to start playback.  If it takes longer than 10
-    // seconds, fail the test.
-    await waiter.waitForMovementOrFailOnTimeout(video, 10);
-
-    // Play for 15 seconds, but stop early if the video ends.  If it takes
-    // longer than 45 seconds, fail the test.
-    await waiter.waitUntilPlayheadReachesOrFailOnTimeout(video, 15, 45);
-
-    await player.unload();
-  });
+    it('raw MP3', async () => {
+      if (!MediaSource.isTypeSupported('audio/mp4; codecs="mp3"')) {
+        return;
```

Comment:
It's better to call Jasmine's `pending()` (terrible name) to "skip" the test, rather than to "pass" it without doing anything.

Same on the others below.